junit-jupiter-api/src/main/java/org/junit/jupiter/api/ClassOrderer.java

```diff
@@ -61,6 +62,22 @@
 @API(status = EXPERIMENTAL, since = "5.8")
 public interface ClassOrderer {
 
+	/**
+	 * Property name used to set the default class orderer class name: {@value}
+	 *
+	 * <h4>Supported Values</h4>
+	 *
+	 * <p>Supported values include fully qualified class names for types that
+	 * implement {@link org.junit.jupiter.api.ClassOrderer}.
+	 *
+	 * <p>If not specified, test classes are not ordered unless test classes are
+	 * annotated with {@link TestClassOrder @TestClassOrder}.
+	 *
+	 * @since 5.8
```

Comment:
`since 5.9`

Reply:
`@since` denotes when the feature was introduced.

`@API(since = ...)` denotes "since" when the feature was assigned the API _status_.

Reply:
So, it's correct as-is: introduced in 5.8 and stable since 5.9.